panoply/constants.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "3.0.1"
+__version__ = "3.0.2"
```

Comment:
New features mean minor update! 🙃

```suggestion
__version__ = "3.1.0"
```